xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/test/java/org/xwiki/rendering/macro/descriptor/DefaultMacroDescriptorTest.java

```diff
@@ -244,4 +250,17 @@ public void testParameterDescriptorWithDescriptionAndMandatoryOnSetter()
         Assert.assertSame(boolean.class, param3Descriptor.getParameterType());
         Assert.assertEquals(true, param3Descriptor.isMandatory());
     }
+
+    @Test
+    public void testLegacyParameterDescriptor()
```

Comment:
We shouldn't prefix with "test", see https://dev.xwiki.org/xwiki/bin/view/Community/Testing/JavaUnitTesting/#HBestpractices

Reply:
I don't understand what is legacy, I don't see a deprecated  parameter descriptor anywhere? I think the test name should be improved or there should be comments.

Reply:
ah I know what's wrong. The name should correspond to the method tested....